therapy/cli.py

```diff
@@ -79,6 +79,8 @@ def update_normalizer_db(normalizer: str, aws_instance: str, db_url: str,
             environ[AWS_ENV_VAR_NAME] = aws_env_name
             confirm_aws_db_use(aws_env_name.upper())
             environ[SKIP_AWS_DB_ENV_NAME] = "true"  # this is already checked above
+            endpoint_url = ""
+            db = Database()
```

Comment:
You could either delete this + move line 91 outside the else condition or you can change line 83 to `db = Database(db_url=endpoint_url)`